### PR TITLE
website: troubleshooting guide update

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### CODE REVIEW

**Comment by deekay2310** [comment]
Screen Shot 2022-07-01 at 3 37 19 PM

File: website/docs/troubleshooting.md [hunk 1]
```diff
@@ -45,6 +45,30 @@ Documents: https://docs.podman.io
...
+```
+
+This shall display a list of active distributions i.e. "my-machine" in this case.
+
+Then,
+
+```sh
+wsl --unregister my-machine
+```
+(Replacing ```my-machine``` with the name that is displayed under ```wsl --list``` for your Podman Machine)
```
Comment (benoitf) [inline website/docs/troubleshooting.md]:
```suggestion
(Replacing `my-machine` with the name that is displayed under `wsl --list` for your Podman Machine)
```

File: website/docs/troubleshooting.md [hunk 2]
```diff
@@ -45,6 +45,30 @@ Documents: https://docs.podman.io
...
+
+It is highly unlikely that you may be stuck in such a situation but if you are, here's a quick fix for it.
+
+Assuming the name of the Podman Machine to be **"my-machine"**, run the following commands in the terminal:
+
+```sh
+wsl --list
+```
+
+This shall display a list of active distributions i.e. "my-machine" in this case.
```
Comment (benoitf) [inline website/docs/troubleshooting.md]:
```suggestion
This shall display a list of active distributions i.e. `my-machine` in this case.
```

File: website/docs/troubleshooting.md [hunk 3]
```diff
@@ -45,6 +45,30 @@ Documents: https://docs.podman.io
...
 ```
 
+### Fixing corrupted Podman Machine in Windows
+
+If at all you are not able to stop your Podman Machine, you will find such an error in the Logs-
+```Error: Error stopping sysd: exit status 1```
+
+It is highly unlikely that you may be stuck in such a situation but if you are, here's a quick fix for it.
+
+Assuming the name of the Podman Machine to be **"my-machine"**, run the following commands in the terminal:
```
Comment (benoitf) [inline website/docs/troubleshooting.md]:
```suggestion
Assuming the name of the Podman Machine to be `my-machine`, run the following commands in the terminal:
```

**Review by benoitf — APPROVED** [review]
👍

**Pull request merged**